recipes/serial/all/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.5)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# Work-around for https://github.com/wjwwood/serial/issues/135
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
```

Comment:
Could you replace this with the content of the separate file?

Reply:
The idea was to make sure this line "works": https://github.com/wjwwood/serial/blob/master/CMakeLists.txt#L5
But you're right, it should be possible to inline it and set `catkin_FOUND` manually.
I will try that.

Reply:
No, I can't seem to trick CMake.
The `find_package(catkin REQUIRED)` fails even if I set `set(catkin_FOUND true)`...

Reply:
this line seems to be useless.

Reply:
Honestly I would just patch upstream CMakeLists, the last release is 6 years old.

Reply:
Maybe... I think the Findcatkin.cmake is easier though.
And the installation can also be fixed without a patch. Like this: https://github.com/conan-io/conan-center-index/commit/f0869cffa133f40cded38b8bab4bf859e2c176d5#diff-196a5863890619f16fe188fdc1392c8fffe603fa16a60e515ba5211615ae54d9R12

I also want to add that I hope that the upstream repo will create a new release.
As you say the latest release is 6 years old, but the latest commit is just 1 years old :)
If I don't get a response we could maybe add a new version like `1.2.2-cbcca7c8`?
Download using: https://github.com/wjwwood/serial/archive/cbcca7c8.tar.gz

Reply:
By fixing installation, I meant install destination of dll.

Reply:
Yes, me too. By adding a new install command it's fixed.
That was the plan at least.

Reply:
```cmake
add_subdirectory(source_subfolder)

install(TARGETS serial)
```